drivers/sensor/ams/ams_as5600/Kconfig

```diff
@@ -1,12 +1,172 @@
 # AS5600 Angular position sensor configuration option
 
-# Copyright (c) 2022, Felipe Neves.
+# Copyright (c) 2025, Cameron Ewing
 # SPDX-License-Identifier: Apache-2.0
 
-config AMS_AS5600
-	bool "AS5600 Angular position sensor"
-	default y
-	depends on DT_HAS_AMS_AS5600_ENABLED
-	select I2C
-	help
-	  Enable driver for AS5600 Angular position sensor.
+menuconfig AMS_AS5600
+    bool "AS5600 Angular position sensor"
+    default y
+    depends on DT_HAS_AMS_AS5600_ENABLED
+    select I2C
+    help
+      Enable driver for AS5600 Angular position sensor.
+
+if  AMS_AS5600
+
+config AS5600_START_POSITION
+    int "Start position"
+    default 0
+    range 0 4095
+    help
+        Start position (min. 0; max. 4095)
+        Difference between Start and End position must be at least 18 degrees (205 steps).
+config AS5600_END_POSITION
+    int "End position"
+    default 0
+    range 0 4095
+    help
+        End position (min. 0; max. 4095)
+        Difference between Start and End position must be at least 18 degrees (205 steps).
+
+config AS5600_ANGULAR_RANGE
+    int "Maximum angular range"
+    range 205 4096
+    default 4096
+    help
+        Maximum angular range (min. 205; max. 4096)
+        The minimum angular range is 18 degrees (205 steps).
```

Comment:
All of these don't seem to belong in Kconfig, what if I wanted 2 of these sensors with different ranges? I think they belong in devicetree.